ImP/preProcessing/preProcessor.py

```diff
@@ -0,0 +1,25 @@
+import cv2 
+import numpy as np
+
+class PreProcessor:
+
+	def __init__(self, pathToNewImage):
+		self.image = cv2.imread(pathToNewImage)
+
+	def is_similar(self, pathToOtherImage):
+		existingImage = cv2.imread(pathToOtherImage)
+		difference = cv2.subtract(self.image, existingImage)
+		if(np.mean(difference) <= 2.2):
```

Comment:
I would move the threshold value (2.2) either to be a class variable, or some kind of "Settings" variable.